hypervisor/lazy.go

```diff
@@ -49,6 +49,13 @@ func statePreparing(ctx *VmContext, ev VmEvent) {
 		ctx.setWindowSize(cmd.ClientTag, cmd.Size)
 	case COMMAND_RUN_POD, COMMAND_REPLACE_POD:
 		glog.Info("got spec, prepare devices")
+		err := ctx.DCtx.(LazyDriverContext).InitVM(ctx)
```

Comment:
The `statePreparing` is called in `LazyVmLoop` function, and `statePreparing` will prepare the VM and devices for COMMAND_RUN_POD and COMMAND_REPLA event.

The `InitVM` and `lazyPrepareDevice` should be together before starting the vm.  We should not divide following process into two function.

```
create vm ---> prepare devices(block devices and nic) ---> start vm
```